README.md

```diff
@@ -174,31 +170,49 @@ interface MerchantRateDetail {
 The `generateVanity` function converts a URL (to a product, listing, etc. on a supported domain) to a `wild.link` URL with embedded tracking for the given device.
 
 ```js
-// entry from getDomains() that matches the domain to be converted
-const domain =   {
+// example Eligible Domain from getDomains() that matches the url to be converted
+{
   ID: 1991737,
-  Domain: "target.com",
+  Domain: 'target.com',
   Merchant: {
     ID: 5482877,
-    Name: "Target",
+    Name: 'Target',
     DefaultRate: {
-      Kind: "PERCENTAGE",
-      Amount: "0.5",
+      Kind: 'PERCENTAGE',
+      Amount: '0.5',
     },
     DerivedRate: {
-      Kind: "PERCENTAGE",
-      Amount: "0.75",
+      Kind: 'PERCENTAGE',
+      Amount: '0.75',
     },
     MaxRate: {
-      Kind: "PERCENTAGE",
-      Amount: "0.5",
+      Kind: 'PERCENTAGE',
+      Amount: '0.5',
+    },
+  },
+};
+
+const url = 'https://www.nixon.com/us/en/gamma-backpack/C3024-000-00.html#start=1';
+
+WLClient.getDomains().then(domains => {
+  for (let i = 0; i < domains.length; i++) {
+    // find the Eligible Domain that matches the domain.tld of the url
+    if (parseUrl(url) === domains[i].Domain) {
```

Comment:
Clarify that parseUrl comes from an external dependency. With a possible link to npm, above the code example.